packages/backend-api/integration/build.xml.patch

```diff
@@ -0,0 +1,33 @@
+@@ -5,10 +5,32 @@
+ 
+     <property name="path.root" value="${project.basedir}"/>
+     <property name="path.vendor" value="${path.root}/vendor"/>
++    <property name="path.backend-api" value="${path.vendor}/backend-api"/>
```

Comment:
i dont think so. please see path.framework it is inside `shopsys/framework` so this will be inside `shopsys/backend-api`